### DISCUSSION TITLE
Report to download for CITES/T&E?

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#4891] [General]
I need to do a search on CITES and T&E species by MSB collection, and I don't see that option in the Search pane.
Is there a SQL query that will pull this?
I can use the Legal option in Manage in Search Results, but this doesn't just give only the ones that are CITES etc - it creates a huge list of everything in the collection, and there is no download. 
Alternately, perhaps this could be an option in See Results As on the Search page?
Once I have the number of cataloged items by CITES/T&E by collection, I also need the parts/table download or a summary by number of cryovials.
Priority - high. Needed for grant proposal.

**Comment by dustymc** [comment]
I think there's a webinar (??) on the report, it's in manage.

The UI is limited (because https://github.com/ArctosDB/arctos/issues/3311) but you can search by building URLs. For example

```
https://arctos.database.museum/SpecimenResults.cfm?tax_rnk_1=CITES%20Appendix&tax_trm_1=I&tax_opr_1=is&tax_src_1=Arctos+Legal&guid_prefix=MSB:Mamm
```
* tax_rnk_1=CITES Appendix - the term type
* tax_trm_1=I - the term
* tax_opr_1=is - exact match of the term
* tax_src_1=Arctos Legal - the source
* guid_prefix=MSB:Mamm - the collection

finds https://arctos.database.museum/guid/MSB:Mamm:101210 because

https://arctos.database.museum/name/Mustela%20nigripes#ArctosLegal

contains

* CITES Appendix: I 

(If you can find it, [ugh](https://github.com/ArctosDB/arctos/issues/3973)....)

<img width="1183" alt="Screen Shot 2022-05-23 at 8 56 19 AM" src="https://user-images.githubusercontent.com/5720791/169859538-b47967c5-cac1-4b29-93de-d92e01a04843.png">

**Comment by campmlc** [comment]
That works, but it's pretty awful.
I know how to use the manage tools for CITES - it's that tool that I was
hoping could be modified to allow a csv download, and to limit the
results/download to only those taxa that are in ARCTOS LEGAL.

**Comment by campmlc** [comment]
@ccicero 
Currently, the search results produces a list of EVERYTHING in the collection, with a little flag as to whether it is CITES or not, and you have to scroll down through the entire list in your browser. There is no download tool. I'm asking for the search results to include only those taxa that are listed in Arctos Legal as CITES or TE in some way, and I'd like to have a download option so that I don't have to just view the results through my browser.

**Comment by wellerjes** [comment]
I am in favor of a report or a flag that tells you which specimens in your collection have a classification in Arctos Legal. I agree with @campmlc, a csv download option would be useful as well. Anything to help narrow down/more easily search for these specimens so we can make sure there are the proper encumbrances on locations, etc.

**Comment by wellerjes** [comment]
Circling back to this because we're working on permits for the year. Is there a way to pull this? How up-to-date is Arctos Legal, and how can I search for specimens in our collection using it? Could we get a handbook page explaining the process?

Thanks!
